doc/engineering-reference/src/alternative-modeling-processes/hybrid-model.tex

```diff
@@ -0,0 +1,187 @@
+\section{Hybrid Model}\label{hybrid-model}
+
+\subsection{Overview}\label{overview-004}
+
+Nomenclature\\
+$Q_{int}$	Internal heat gain\\
+$h_s$	Convective heat transfer coefficient\\
+$A_s$	Zone surface area\\
+$T_s$	Zone surface temperature\\
+$T_z$	Zone air temperature\\
+$T_{iz}$	Interzone air temperature\\
+$T_o$	Outdoor air temperature\\
+$T_{sup}$	Air system supply air temperature\\
+$q_{inf}$	Infiltration air flow rate\\
+$m ̇_i$z	Interzone air mass flow rate\\
+$m ̇_{sys}$	Air system mass flow rate\\
+$V$	Zone volume\\
+$ρ_{air}$	Air density\\
+$C_z$	Heat capacity of zone air and internal thermal mass\\
+$C_p$	Zone air specific heat\\
+$C_T$	Heat capacity multiplier\\
+
+The hybrid approach aims to enhance the current energy retrofit practices not only offering more user-friendly energy modeling environments but also providing more accurate estimates of energy savings at the same time. The hybrid model avoids input parameters that are hard to measure and replace them to easily measurable data, which integrates physics-based and data-driven modeling methods. Parameters such as interior thermal mass and air infiltration rates are required in physics-based models, and they have significant impacts as they are driving factors for the dynamic performance of buildings. An accurate estimate of interior thermal mass has been a difficult problem because the building usually has various amounts of furniture and changeable partitions. The air infiltration rate changes in time and dynamically interacts with indoor and outdoor climatic conditions. However the accurate estimation of the data is almost impossible to collect without a fan pressurized test, which can not be easily done by typical energy modelers (Gowri et al. 2009). The hybrid model introduces an approach that represents the interior thermal mass in EnergyPlus, one with traditional interior mass object and the other with capacitance multiplier approach. Also the approach presents fundamentals of the infiltration modeling in energy simulation and provides mathematical derivation of the hybrid model. 
```

Comment:
Updated the beginning section adding the hybrid model definition for the feature:

The hybrid modeling combines forward and inverse physics-based modeling taking advantage of measured zone air temperature data to perform targeted calibration on specific inputs.